app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityController.scala

```diff
@@ -1459,15 +1459,29 @@ class YearToDateLiabilityController @Inject() (
                   case Some(yearToDateLiability) =>
                     if (nonCalculatedAnswers.fold(_.yearToDateLiability, _.yearToDateLiability).isEmpty)
                       Redirect(routes.YearToDateLiabilityController.yearToDateLiability())
-                    else
+                    else {
+                      val taxOwnedOnOriginalReturn = fillingOutReturn.amendReturnData
```

Comment:
it makes more sense to talk about `taxOwed` rather than `taxOwned`

Reply:
Yes typo error, will fix it everywhere